man/ocitools-generate.1.md

```diff
@@ -234,6 +234,45 @@ inside of the container.
   The special *PATH* `host` removes any existing UTS namespace from
   the configuration.
 
+**--cpu-shares**=CPUSHARES
+  Sets the CPU shares (relative weight (ratio) vs. other cgroups with cpu shares).
```

Comment:
@wking , thanks for pointing out.
In fact, the current description I am using is from https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/specs-go/config.go#L272 . 
Is there any plan on runtime-spec side to sync the descriptions on the config-linux.md and config.go?